api/src/main/java/com/codeforcommunity/dto/report/ReportWithLibrary.java

```diff
@@ -122,9 +126,10 @@ public ReportWithLibrary(
       String actionPlan,
       String successStories,
       List<Grade> gradesAttended,
-      String timetable,
+      String checkInTimetable,
```

Comment:
can we regroup these arguments so checkin and checkout timeable are together?

Reply:
By this do you mean like have it be in a class itself that holds these two fields?

Reply:
No I just meant that its weird that the two timetable arguments arent together in the argument list, I think they should be grouped together